tests/platform_helper/providers/test_config_validator.py

```diff
@@ -286,12 +287,164 @@ def test_validate_database_copy_fails_if_cross_account_with_incorrect_account_id
     with pytest.raises(SystemExit):
         ConfigValidator().validate_database_copy_section(config)
 
-    console_message = capfd.readouterr().err
+    console_message = capsys.readouterr().err
 
     msg = f"Incorrect value for 'from_account' for environment 'prod'"
     assert msg in console_message
 
 
+def test_validate_platform_config_fails_if_database_copy_config_is_invalid(
```

Comment:
Maybe...
```suggestion
def test_validate_platform_config_fails_if_database_copy_to_and_from_are_the_same(
```

Reply:
Or if we are not specifically testing this misconfiguration, perhaps asserting on the message is not needed here?